src/main/js/helpers/axis.js

```diff
@@ -249,6 +267,74 @@ const processTickValues = (ticks) => {
     }
     return ticks.map((t) => (utils.isDate(t) ? utils.parseDateTime(t) : t));
 };
+
+/**
+ * Gets the number of ticks on the axis based on the upper and lower limits
+ *
+ * @private
+ * @param {number} range - range of values (upperLimit - lowerLimits)
+ * @returns {number} returns number of ticks for that range, based on a predefined set
+ */
+const getTicksCountFromRange = (range) => {
+    if (range <= constants.AXISRANGE_1) return constants.DEFAULT_TICKSCOUNT - 4;
+    if (range <= constants.AXISRANGE_2) return constants.DEFAULT_TICKSCOUNT - 3;
+    if (range <= constants.AXISRANGE_3) return constants.DEFAULT_TICKSCOUNT - 2;
```

Comment:
Can we use the `switch` statement here?

Reply:
[Fixed](https://github.com/cerner/carbon-graphs/pull/216/commits/62d2f2e7657e0d5ff36db3bbfb750a728bc22000)